### DISCUSSION TITLE
Drag Symbol after LongPress

### DISCUSSION THREAD

**Discussion by danieljosua1** [maplibre/flutter-maplibre-gl#363] [Q&A]
Hey guys, i got following question:
I have several symbols on the map that can only be dragged if a LongPress has taken place on them. I want to prevent users from accidentally moving the symbol when panning. I am currently updating the respective symbol after a longpress with the updateSymbol function so that it becomes draggable. However, for this approach the finger has to leave the screen briefly so that the updated symbol can be dragged. Is it somehow possible to enable this process in a single gesture (LongPress->Drag Symbol)?
Thanks in advance for any hints!

**Comment by danieljosua1** [comment]
It is resolved for me. I have written my own implementation which is laid over Maplibre.

**Reply by m0nac0** [reply]
That's good to hear! If you're able to share your approach or implementation, you're very welcome to do so (e.g. paste the code or link to a repo/gist), maybe someone else will have the same issue.
